src/accel/RunAction.cc

```diff
@@ -0,0 +1,86 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2022 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file accel/RunAction.cc
+//---------------------------------------------------------------------------//
+#include "RunAction.hh"
+
+#include <CLHEP/Random/Random.h>
```

Comment:
Why is this header necessary? And same comments for G4TransporationManager.hh and all other celeritas headers below.  Are they included on purpose for linking tests?

Reply:
It's an IWYU error on my part when I (temporarily) deleted the physics construction in https://github.com/celeritas-project/celeritas/commit/3059cf04b165091edfb857378ae0847c2863eee6 . I'll run include-what-you-use and then Amanda can restore the headers in her followup branch.